cloud/scope/powervs_cluster.go

```diff
@@ -1977,3 +1969,26 @@ func (s *PowerVSClusterScope) isResourceCreatedByController(resourceType infrav1
 	}
 	return false
 }
+
+func (s *PowerVSClusterScope) bucketName() string {
```

Comment:
can't this be moved under GetServiceName() method? we can create a resource type as bucket .

Reply:
Done.